src/ensembl/src/content/app/browser/drawer/drawer-views/transcript-summary/TranscriptSummary.tsx

```diff
@@ -14,70 +14,323 @@
+
+      <div className={`${styles.row} ${styles.spaceAbove}`}>
+        <div className={styles.label}>Transcript name</div>
+        <div className={styles.value}>{transcript.symbol}</div>
+      </div>
+
+      <div className={styles.row}>
+        <div className={styles.label}>Transcript length</div>
+        <div className={styles.value}>
+          {getCommaSeparatedNumber(transcript.slice.location.length)}{' '}
+          <span className={styles.unit}>bp</span>{' '}
+        </div>
+      </div>
+
+      <div className={styles.row}>
+        <div className={styles.value}>
+          {transcript.spliced_exons.length} exons
+        </div>
+      </div>
+
+      <div className={`${styles.row} ${styles.spaceAbove}`}>
+        <div className={styles.label}>Coding exons</div>
+        <div className={styles.value}>{getNumberOfCodingExons(transcript)}</div>
+      </div>
+
+      <div className={styles.row}>
+        <div className={styles.label}>Combined exon length </div>
+        <div className={styles.value}>
+          {splicedRNALength} <span className={styles.unit}>bp</span>
+        </div>
+      </div>
+
+      {transcript.so_term === 'protein_coding' && (
+        <div className={styles.row}>
+          <div className={styles.label}>Protein</div>
+          <div className={styles.value}>
+            <div>
+              {productAminoAcidLength} <span className={styles.unit}>aa</span>
+            </div>
+            {product && (
+              <div className={styles.lightText}>{product.stable_id}</div>
+            )}
+            {uniprotXref && (
+              <ExternalReference
+                classNames={{ label: styles.lightText }}
+                label={'UniProtKB/Swiss-Prot'}
+                to={uniprotXref.url}
+                linkText={uniprotXref.accession_id}
+              />
+            )}
+          </div>
+        </div>
+      )}
+
+      {ccdsXref && (
+        <div className={`${styles.row} ${styles.spaceAbove}`}>
+          <div className={styles.value}>
+            <ExternalReference
+              classNames={{ label: styles.lightText }}
+              label={'CCDS'}
+              to={ccdsXref.url}
+              linkText={ccdsXref.accession_id}
+            />
+          </div>
+        </div>
+      )}
+
+      <div className={`${styles.row} ${styles.spaceAbove}`}>
+        <div className={styles.value}>
+          <span
+            className={styles.downloadLink}
+            onClick={() => showDownload(!shouldShowDownload)}
+          >
+            Download
           </span>
+          {shouldShowDownload && (
+            <div className={styles.downloadWrapper}>
+              <InstantDownloadTranscript
+                genomeId={ensObjectGene.genome_id}
+                transcript={{
+                  id: transcript.unversioned_stable_id,
+                  so_term: transcript.so_term
+                }}
+                gene={{ id: gene.unversioned_stable_id }}
+                theme={'light'}
```

Comment:
you don't need the interpolation if the property is a string (`theme="light"`)

Reply:
That's done